components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -1169,7 +1169,7 @@ private Condition createConditionForSingleAttributeFilter(String domainName, Nod
         try {
             if (SCIMConstants.UserSchemaConstants.GROUP_URI.equals(attributeName)) {
                 if (carbonUM instanceof AbstractUserStoreManager) {
-                    String[] roleNames = getRoleNames(filterOperation, attributeValue);
+                    String[] roleNames = getRoleNames(attributeName, filterOperation, attributeValue);
```

Comment:
Better to update the signature in order below
filterOperation
attributeName
attributeValue

Reply:
To maintain the consistency, parameters were set in the attributeName, filterOperation and attributeValue order